dev-docs/source/Testing/SANSGUI/SANSGUITests.rst

```diff
@@ -9,101 +9,183 @@ SANS GUI Testing
 Data reduction
 --------------
 
-*Preparation*
+**Time required: about 1 hour**
 
--  Get the ISIS sample data from the website
--  Ensure that the ISIS sample data directory is in the search directories
+--------------
 
+Set up
+######
 
-**Time required 20 - 30 minutes**
+#. Get the Training data from the downloads page on the Mantid Project website.
+#. Open ``Interfaces`` > ``SANS`` > ``SANS v2``.
+#. Click ``Manage Directories``. This opens Mantid's main dialog for managing
+   search paths. Ensure the Training data directory is in the search
+   directories and that the default save path is set..
+#. Close the manage-directories dialog and ensure the ``Save Path`` on the GUI
+   displays the default save path.
+#. Click ``Load User File``; from the Training data in the ``loqdemo`` folder,
+   choose ``MaskFile.toml``.
+#. Click ``Load Batch File``; from the Training data in the ``loqdemo`` folder,
+   choose ``batch_mode_reduction.csv``.
 
---------------
+Runs table editing
+##################
+
+In the ``Runs`` tab:
+
+#. Check that the ``Insert``, ``Delete``, ``Copy``, ``Paste``, ``Cut`` and
+   ``Erase`` icons work as expected on table rows.
+#. Tick the ``Sample Geometry`` button - some extra columns should appear.
+#. Create multiple rows in the table with different data e.g. one row with
+   an output name, one row without.
+#. Click the ``Export Table`` button and save the table as a csv file. Check
+   the file in an editor or Excel and ensure it looks like a sensible
+   representation of the table in the format ``key,value,key,value,...``. All
+   columns except ``Options`` and ``Sample Shape`` should be included.
+#. Try unticking ``Sample Geometry`` and ticking ``Multi-period``. The
+   displayed columns change but the saved file should contain the same set of
+   columns regardless of whether these are ticked.
+#. Click ``Load Batch file`` and select the newly saved table. All columns
+   that were saved should be loaded.
+#. Try deleting and/or reordering some of the columns in the saved file and
```

Comment:
Ahh sorry I've just realised what section I posted this under.

I mean delete / reorganise the table in the GUI (rather than the exported file) should not reset the settings in the GUI